src/time_integration/low_storage_integrator.cpp

```diff
@@ -128,31 +140,38 @@ LowStorageIntegrator::LowStorageIntegrator(const std::string &name)
     beta.resize(nstages);
     gam0.resize(nstages);
     gam1.resize(nstages);
+    c.resize(nstages);
 
     delta[0] = 1.0;
     beta[0] = 0.357534921136978;
     gam0[0] = 0.0;
     gam1[0] = 1.0;
+    c[0] = 0.0;
 
     delta[1] = 0.0;
     beta[1] = 2.364680399061355;
     gam0[1] = -3.666545952121251;
     gam1[1] = 4.666545952121251;
+    c[1] = 0.357534921136978;
 
     delta[2] = 0.0;
     beta[2] = 0.016239790859612;
     gam0[2] = 0.035802535958088;
     gam1[2] = 0.964197464041912;
+    c[2] = 1.0537621812245777;
```

Comment:
Is a time offset >1 allowed?

Reply:
That is unexpected.  I observed the expected order of convergence so I assumed it was correct but am not an expert in SSP RK methods.

Reply:
I couldn't find any restrictions `c` so I experimented with changing `c[2] <- 1.0` and the result was a loss of accuracy (to 1st order).